docs/linter.md

```diff
@@ -0,0 +1,38 @@
+# Linter
+
+The Bicep linter will inspect your code and catch a customizable set of authoring best practices. It will surface warnings, errors, or informational messages as you're typing in VS Code for immediate feedback. This means you don't have to build or deploy your code to find out that you've made a mistake. Some rules can also surface an automatic code fix through the VS Code light bulb. 
+
+The linter should make it easier to enforce team coding standards by providing guidance during the development inner loop, as well as the ability to break a build (if desired) during continuous integration (CI) for violations.
+
+![linter demo](./images/linter.gif)
+
+## Configuration
+
+[`bicepsettings.json`](./src/Bicep.Core/Configuration/bicepsettings.json) can be used to:
```

Comment:
Is the linked bicepsettings file "complete" in the sense that it shows all options? It might be good to have part of this doc cover the schema of this JSON file in more depth